core/templates/dev/head/admin/Admin.js

```diff
@@ -56,10 +56,10 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
     $http.post($scope.adminHandlerUrl, {
       action: 'revert_config_property',
       config_property_id: configPropertyId
-    }).success(function() {
+    }).then(function() {
       $scope.message = 'Config property reverted successfully.';
       $scope.reloadConfigProperties();
-    }).error(function(errorResponse) {
+    }, function(errorResponse) {
       $scope.message = 'Server error: ' + errorResponse.error;
```

Comment:
I don't think we add a custom .error, but this should be uncovered via testing. The easiest way to induce an error from the server is to go to core.controllers.admin and add a line in the relevant handler that raises an Exception -- this should result in a 500.